authapi/tests/test_organizations.py

```diff
@@ -211,6 +235,153 @@ def test_summary_serializer(self):
+
+        # Authenticated request, correct permissions
+        self.add_permission(user, 'org:admin')
+        response = self.client.post(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Admin user
+        _, token = self.create_admin_user()
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+        response = self.client.post(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_permission_get_organization(self):
+        '''Any authenticated user should be able to get the details of an
+        organization.'''
+        org = SeedOrganization.objects.create()
+        url = reverse('seedorganization-detail', args=(org.pk,))
+
+        # Unauthenticated request
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+        # Authenticated request
+        _, token = self.create_user()
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_permission_update_organization(self):
+        '''Only admin users, users with org:admin permissions, and users with
+        org:write permissions for the organization should be able to update
+        organizations.'''
+        data = {
+            'title': 'test org',
+        }
+        org1 = SeedOrganization.objects.create()
+        org2 = SeedOrganization.objects.create()
+        url = reverse('seedorganization-detail', args=(org1.pk,))
+
+        # Unauthenticated request
+        response = self.client.put(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+        # Authenticated request, no permissions
+        SeedPermission.objects.all().delete()
+        user, token = self.create_user()
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + token.key)
+        response = self.client.put(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+        # Authenticated request, wrong org permissions
+        SeedPermission.objects.all().delete()
+        self.add_permission(user, 'org:write', org2.pk)
+        response = self.client.put(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+        # Authenticated request, correct org permissions
+        SeedPermission.objects.all().delete()
+        self.add_permission(user, 'org:write', org1.pk)
+        response = self.client.put(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+        # Authenticated request, org:admin permissions
+        SeedPermission.objects.all().delete()
+        self.add_permission(user, 'org:admin')
+        response = self.client.put(url, data=data)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
Do we need a test here for wrong org for `org:admin`? Similarly for other methods' tests.